app/models/skin.rb

```diff
@@ -48,7 +48,7 @@ class Skin < ApplicationRecord
                                   class_name: 'SkinParent', dependent: :destroy, inverse_of: :parent_skin
   has_many :child_skins, through: :skin_children, inverse_of: :parent_skins
 
-  accepts_nested_attributes_for :skin_parents, allow_destroy: true, reject_if: proc { |attrs| attrs[:position].blank? }
+  accepts_nested_attributes_for :skin_parents, allow_destroy: true, reject_if: proc { |attrs| attrs[:position].blank? || attrs[:title].blank? }
```

Comment:
I'm surprised that checking `title` here would work -- I would have thought that you'd have to check `parent_skin_title`, because that's the name in the permitted params:
https://github.com/otwcode/otwarchive/blob/808b34b1d21be65c26fc9ae4a20621c40955a37d/app/controllers/skins_controller.rb#L176-L178
Have you played around with adding parent skins in development to make sure that it's not rejecting all parents, instead of just blank ones?

Reply:
Oh I missed that attributes listing, thank you! You're right that it was rejecting all parents, I missed that and added it to my own testing. Changing to parent_skin_title does appear to have fixed that.